ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -187,6 +187,13 @@ if tflite2_support_is_available
   tflite2_compile_args += '-DTFLITE_FLOAT16=1'
   tflite2_compile_args += '-DTFLITE_COMPLEX64=1'
 
+  if get_option('gpu-delegate-support').enabled()
+    # GLES dependency for tflite GPU delegate
+    gles_dep = dependency('gles20', required: true)
```

Comment:
how about adding this in features (https://github.com/nnstreamer/nnstreamer/blob/b7d63427b7da95b9453a50f4eac96c5c4822431a/meson.build#L242)

  'tflite2-gpu-delegate-support': {
    'target': 'gles20',
    'project_args': { 'TFLITE_GPU_DELEGATE_SUPPORTED': 1 }
  },

Reply:
Fixed.